core/site.go

```diff
@@ -101,7 +101,7 @@ type Site struct {
 	pvPower       float64         // PV power
 	excessDCPower float64         // PV excess DC charge power (hybrid only)
 	auxPower      float64         // Aux power
-	batteryPower  float64         // Battery charge power
+	batteryPower  float64         // Battery discharge power
```

Comment:
```suggestion
	batteryPower  float64         // Battery power
```

It's neither nor

Reply:
a positive value means that energy is leaving the battery, doesn't it?

Reply:
i have now entered a negative value into `cmd/demo.yaml` and the battery's charge is increasing as a result of that

Reply:
Yes, this certainly clarifies the meaning :)